odl/solvers/nonsmooth/primal_dual_hybrid_gradient.py

```diff
@@ -196,14 +203,24 @@ def chambolle_pock_solver(x, f, g, L, tau, sigma, niter, **kwargs):
         raise ValueError('`theta` {} not in [0, 1]'
                          ''.format(theta_in))
 
-    # Acceleration parameter
-    gamma = kwargs.pop('gamma', None)
-    if gamma is not None:
-        gamma, gamma_in = float(gamma), gamma
-        if gamma < 0:
-            raise ValueError('`gamma` must be non-negative, got {}'
-                             ''.format(gamma_in))
-
+    # Acceleration parameters
+    gamma_primal = kwargs.pop('gamma_primal', None)
+    if gamma_primal is not None:
+        gamma_primal, gamma_primal_in = float(gamma_primal), gamma_primal
+        if gamma_primal < 0:
+            raise ValueError('`gamma_primal` must be non-negative, got {}'
+                             ''.format(gamma_primal_in))
+
+    gamma_dual = kwargs.pop('gamma_dual', None)
+    if gamma_dual is not None:
+        gamma_dual, gamma_dual_in = float(gamma_dual), gamma_dual
+        if gamma_dual < 0:
+            raise ValueError('`gamma_dual` must be non-negative, got {}'
+                             ''.format(gamma_dual_in))
+
+    if gamma_primal is not None and gamma_dual is not None:
+        raise ValueError('Only one acceleration parameter can be used')
```

Comment:
Done. Depending on which part is strongly convex, the step sizes get either multiplied or divided by theta. Both doesn't make sense.